src/pages/core/guides/anchor/how-to-create-a-core-collection.md

```diff
@@ -0,0 +1,440 @@
+---
+title: How to Create a Core Collection
+metaTitle: How to Create a Core Collection | Core Guides
```

Comment:
SEO wise we should not use the same titles twice, so you might want to add "using Anchor" here and "using JavaScript" in the other file.